website/docs/reference/resource-configs/target_database.md

```diff
@@ -4,6 +4,16 @@ description: "Target_database - Read this in-depth guide to learn about configur
 datatype: string
 ---
 
+<VersionBlock firstVersion="1.9">
+
+:::warning Legacy configuration
+
+For environments on versionless dbt Cloud or dbt Core v1.9+, the `target_database` configuration is now optional. Best practices dictate that the `target_database` config should be removed from snapshots in the environment, in which case the snapshots will instead utilize the `generate_database_name` macro by default. Project snapshots configured with `target_schema` will continue to work as expected. 
+
```

Comment:
I think worth calling out the new `database` config as the alternative way to put snapshots in a custom database while still respecting `generate_database_name` macro

Reply:
💪

Reply:
Done and done. Added similar language to the schema section as well